1.10.15/buster/build-time-pip-constraints.txt

```diff
@@ -175,7 +175,7 @@ typing-inspect==0.6.0
 tzlocal==1.5.1
 unicodecsv==0.14.1
 uritemplate==3.0.1
-urllib3==1.25.11
+urllib3==1.26.8
```

Comment:
I'm confused... circleci sees the new protobuf, but not urllib3? Any ideas?

Reply:
<img width="1379" alt="image" src="https://user-images.githubusercontent.com/8811558/155535634-baec0115-5827-427e-8d4c-4759be93a0c4.png">

>Successfully built Flask-Admin Flask-JWT-Extended Flask-Login Flask-OpenID flask-swagger future grpc-google-iam-v1 json-merge-patch mysqlclient proto-plus psutil pyrsistent pysftp python-nvd3 python-slugify slackclient SQLAlchemy-Utils thrift tornado tzlocal unicodecsv
ERROR: requests 2.23.0 has requirement urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1, but you'll have urllib3 1.26.8 which is incompatible.
ERROR: elasticsearch-dsl 5.4.0 has requirement elasticsearch<6.0.0,>=5.0.0, but you'll have elasticsearch 7.13.4 which is incompatible.
ERROR: botocore 1.13.50 has requirement urllib3<1.26,>=1.20; python_version >= "3.4", but you'll have urllib3 1.26.8 which is incompatible.
Installing collected packages: wrapt, deprecated, zipp, importlib-metadata, pyparsing, packaging, redis, six, python-dateutil, azure-nspkg, azure-common, pycparser, cffi, cryptography, idna, chardet, urllib3, certifi, requests, azure-storage, elasticsearch, PyJWT, adal, SQLAlchemy, MarkupSafe, Mako, python-editor, alembic, vine, amqp, PyYAML, apispec, appdirs, argcomplete, distro, pyrsistent, attrs, jsonschema, WTForms, itsdangerous, Jinja2, click, Werkzeug, Flask, Flask-WTF, Markdown, cattrs, gunicorn, importlib-resources, cached-property, text-unidecode, python-slugify, python-nvd3, dnspython, email-validator, funcsigs, pytz, tzlocal, flask-swagger, future, Pygments, pytzdata, pendulum, setproctitle, marshmallow, marshmallow-sqlalchemy, dill, json-merge-patch, numpy, pandas, tenacity, typing-extensions, unicodecsv, zope.deprecation, Flask-Caching, Flask-Login, configparser, psutil, iso8601, lazy-object-proxy, docutils, lockfile, python-daemon, colorlog, tabulate, graphviz, SQLAlchemy-JSONField, Flask-Admin, colorama, Babel, Flask-Babel, Flask-JWT-Extended, marshmallow-enum, Flask-SQLAlchemy, SQLAlchemy-Utils, prison, defusedxml, python3-openid, Flask-OpenID, Flask-AppBuilder, thrift, natsort, croniter, apache-airflow, greenlet, eventlet, zope.event, zope.interface, gevent, oauthlib, requests-oauthlib, isodate, msrest, azure-core, azure-storage-blob, azure-storage-common, msrestazure, azure-mgmt-containerinstance, azure-cosmos, kombu, billiard, celery, tornado, humanize, prometheus-client, flower, elasticsearch-dsl, jmespath, botocore, s3transfer, boto3, protobuf, pyasn1, pyasn1-modules, rsa, cachetools, google-auth, googleapis-common-protos, grpcio, google-api-core, google-cloud-core, google-cloud-translate, grpc-google-iam-v1, google-cloud-bigtable, google-cloud-dlp, google-crc32c, google-resumable-media, google-cloud-storage, google-cloud-speech, proto-plus, pyarrow, mypy-extensions, typing-inspect, libcst, google-cloud-bigquery-storage, google-cloud-bigquery, google-auth-oauthlib, pydata-google-auth, pandas-gbq, google-cloud-spanner, google-cloud-language, httplib2, google-auth-httplib2, uritemplate, google-api-python-client, google-cloud-secret-manager, google-cloud-texttospeech, google-cloud-videointelligence, google-cloud-vision, pyOpenSSL, google-cloud-container, grpcio-gcp, websocket-client, kubernetes, mysqlclient, psycopg2-binary, slackclient, PyNaCl, bcrypt, paramiko, pysftp, sshtunnel, statsd, distlib, filelock, virtualenv, astronomer-certified, astronomer-airflow-version-check
Successfully installed Babel-2.9.1 Flask-1.1.2 Flask-Admin-1.5.4 Flask-AppBuilder-2.3.4 Flask-Babel-1.0.0 Flask-Caching-1.3.3 Flask-JWT-Extended-3.25.0 Flask-Login-0.4.1 Flask-OpenID-1.2.5 Flask-SQLAlchemy-2.4.4 Flask-WTF-0.14.3 Jinja2-2.11.2 Mako-1.1.3 Markdown-2.6.11 MarkupSafe-1.1.1 PyJWT-1.7.1 PyNaCl-1.4.0 PyYAML-5.4.1 Pygments-2.7.4 SQLAlchemy-1.3.20 SQLAlchemy-JSONField-0.9.0 SQLAlchemy-Utils-0.36.8 WTForms-2.3.3 Werkzeug-0.16.1 adal-1.2.5 alembic-1.4.3 amqp-2.6.1 apache-airflow-1!1.10.15.dev3+astro.5 apispec-1.3.3 appdirs-1.4.4 argcomplete-1.12.2 astronomer-airflow-version-check-1.0.7 astronomer-certified-1.10.15.post5.dev3 attrs-20.3.0 azure-common-1.1.26 azure-core-1.9.0 azure-cosmos-3.2.0 azure-mgmt-containerinstance-1.5.0 azure-nspkg-3.0.2 azure-storage-0.36.0 azure-storage-blob-12.6.0 azure-storage-common-2.1.0 bcrypt-3.2.0 billiard-3.6.3.0 boto3-1.10.50 botocore-1.13.50 cached-property-1.5.2 cachetools-4.1.1 cattrs-1.1.2 celery-4.4.7 certifi-2020.11.8 cffi-1.14.4 chardet-3.0.4 click-6.7 colorama-0.4.4 colorlog-4.0.2 configparser-3.5.3 croniter-0.3.36 cryptography-3.3.2 defusedxml-0.6.0 deprecated-1.2.13 dill-0.3.3 distlib-0.3.1 distro-1.5.0 dnspython-1.16.0 docutils-0.15.2 elasticsearch-7.13.4 elasticsearch-dsl-5.4.0 email-validator-1.1.2 eventlet-0.31.0 filelock-3.0.12 flask-swagger-0.2.14 flower-0.9.5 funcsigs-1.0.2 future-0.18.2 gevent-21.1.2 google-api-core-1.23.0 google-api-python-client-1.12.8 google-auth-1.23.0 google-auth-httplib2-0.0.4 google-auth-oauthlib-0.4.2 google-cloud-bigquery-2.4.0 google-cloud-bigquery-storage-2.1.0 google-cloud-bigtable-1.6.0 google-cloud-container-1.0.1 google-cloud-core-1.4.3 google-cloud-dlp-1.0.0 google-cloud-language-1.3.0 google-cloud-secret-manager-1.0.0 google-cloud-spanner-1.19.1 google-cloud-speech-1.3.2 google-cloud-storage-1.33.0 google-cloud-texttospeech-1.0.1 google-cloud-translate-1.7.0 google-cloud-videointelligence-1.16.1 google-cloud-vision-1.0.0 google-crc32c-1.0.0 google-resumable-media-1.1.0 googleapis-common-protos-1.52.0 graphviz-0.15 greenlet-1.1.0 grpc-google-iam-v1-0.12.3 grpcio-1.33.2 grpcio-gcp-0.2.2 gunicorn-20.0.4 httplib2-0.19.0 humanize-3.1.0 idna-2.8 importlib-metadata-2.1.0 importlib-resources-1.5.0 iso8601-0.1.13 isodate-0.6.0 itsdangerous-1.1.0 jmespath-0.10.0 json-merge-patch-0.2 jsonschema-3.2.0 kombu-4.6.11 kubernetes-11.0.0 lazy-object-proxy-1.4.3 libcst-0.3.14 lockfile-0.12.2 marshmallow-2.21.0 marshmallow-enum-1.5.1 marshmallow-sqlalchemy-0.23.1 msrest-0.6.19 msrestazure-0.6.4 mypy-extensions-0.4.3 mysqlclient-1.3.14 natsort-7.1.0 numpy-1.19.4 oauthlib-3.1.0 packaging-20.7 pandas-1.1.4 pandas-gbq-0.14.1 paramiko-2.7.2 pendulum-1.4.4 prison-0.1.3 prometheus-client-0.8.0 proto-plus-1.11.0 protobuf-3.15.0 psutil-5.7.3 psycopg2-binary-2.8.6 pyOpenSSL-20.0.0 pyarrow-0.17.1 pyasn1-0.4.8 pyasn1-modules-0.2.8 pycparser-2.20 pydata-google-auth-1.1.0 pyparsing-2.4.7 pyrsistent-0.17.3 pysftp-0.2.9 python-daemon-2.2.4 python-dateutil-2.8.1 python-editor-1.0.4 python-nvd3-0.15.0 python-slugify-4.0.1 python3-openid-3.2.0 pytz-2020.4 pytzdata-2020.1 redis-4.1.4 requests-2.23.0 requests-oauthlib-1.3.0 rsa-4.6 s3transfer-0.2.1 setproctitle-1.2 six-1.15.0 slackclient-1.3.2 sshtunnel-0.1.5 statsd-3.3.0 tabulate-0.8.7 tenacity-4.12.0 text-unidecode-1.3 thrift-0.13.0 tornado-5.1.1 typing-extensions-3.7.4.3 typing-inspect-0.6.0 tzlocal-1.5.1 unicodecsv-0.14.1 uritemplate-3.0.1 urllib3-1.26.8 vine-1.3.0 virtualenv-20.2.1 websocket-client-0.54.0 wrapt-1.13.3 zipp-3.4.0 zope.deprecation-4.4.0 zope.event-4.5.0 zope.interface-5.4.0
WARNING: You are using pip version 19.3.1; however, version 22.0.3 is available.

Looks like there are some incompatibilities - https://app.circleci.com/pipelines/github/astronomer/ap-airflow/2552/workflows/17647fe0-2f62-4bef-a788-1dbed3701747/jobs/61576

Reply:
Going to ignore this one as well because urllib3 -> requests -> snowflake. DOS, and lowish risk for us I think.